README.md

```diff
@@ -38,6 +38,79 @@ If you're interested in building tutorials, keep reading!
+
+If you have an idea for a new tutorial that has not yet been proposed, please [open an new issue](https://github.com/ProtoSchool/protoschool.github.io/issues/new?assignees=&labels=new-tutorial&template=tutorial-proposal.md&title=New+Tutorial%3A+%5BProposed+title%5D) so we can share feedback before you get started building. With help from our guiding principles, we'll work together to make sure it fits _cohevisely_ into the ProtoSchool curriculum and will encourage you to create a content outline that supports a _beginner-friendly_ learning experience with appropriate _scaffolding_.
+
+Don't feel like you have the skills or time to build a tutorial yourself? Please still go ahead and share your idea! There may be another community member who can help out.
+
+### Know the capabilities of our platform (and yourself!)
+
+A ProtoSchool tutorial is made up of a series of lessons. We currently support the following lesson types, which can be mixed and matched within a tutorial:
+- Text-only lessons
+- Multiple-choice lessons
+- Code challenges (with or without file upload required)
+
+You'll learn more about how to build each lesson type in our section on [building your lessons](#build-your-lessons-repeat-for-each-lesson-in-the-tutorial).
+
+#### What you need to know to build a tutorial
+
+The text portions of all of our lessons are written in markdown, which is a fairly simple way to style text on the web. [Learn more about markdown formatting here.](https://guides.github.com/features/mastering-markdown/)
+
+Whether or not you know how to code, you should be able to follow the instructions here to build a tutorial made up of **text-only or multiple-choice lessons** by using our boilerplate templates.
+
+To build **code challenges**, you'll need to know JavaScript, both to provide the starter and solution code and to create validation code to evaluate the code submitted by the user.
+
+Although our tutorials and website are hosted within the Vue.js framework, we've done our best to provide enough guidance so that **you won't need any familiarity with Vue.js** to build your own tutorial.
+
+#### Browser-based code challenges
+
+**All code challenges must run in JavaScript in the browser**; we don't currently support tutorials that run from the command line or in other coding languages. We believe that offering our tutorials directly in the browser, as opposed to via the command line, reduces the barrier to entry for our learners. (Do you remember when cloning a repo, installing dependencies, and using the terminal felt overwhelming? We do!)
+
+In the context of lessons on IPFS, this means that our coding challenges teach js-ipfs API methods. You can help our learners who specialize in langauges other than JavaScript by ensuring that you provide links to documentation on any complex JavaScript methods that are required, so that they can focus on learning IPFS rather than JavaScript in your tutorial.
```

Comment:
```suggestion
In the context of lessons on IPFS, this means that our coding challenges teach js-ipfs API methods. You can help our learners who specialize in languages other than JavaScript by ensuring that you provide links to documentation on any complex JavaScript methods that are required, so that they can focus on learning IPFS rather than JavaScript in your tutorial.
```